elm/StalinSort.elm

```diff
@@ -0,0 +1,12 @@
+module StalinSort exposing (stalinsort)
+
+stalinsort : List comparable -> List comparable
+stalinsort list =
+  case list of
+    x::y::zs ->
+      if x < y then
+        x :: stalinsort (y::zs)
+      else
+        stalinsort (y::zs)
+      _ ->
+        list
```

Comment:
For pattern matters, could you add an exporting function?

Reply:
It considers equals numbers?

Reply:
I forgot considers equals numbers.
I fixed it now.